### PR TITLE
Added disable_marker_clustering_at_zoom option

### PR DESCRIPTION
**Pull request by matyasrada**
Exposing disableClusteringAtZoom option #823.

### CODE REVIEW

File: js/src/layers/MarkerCluster.js [hunk 1]
```diff
@@ -63,6 +65,10 @@ export class LeafletMarkerClusterView extends layer.LeafletLayerView {
   }
 
   create_obj() {
-    this.obj = L.markerClusterGroup();
+    var options = this.get_options();
+    options.data = this.model.get('data');
```
Comment (martinRenou) [inline js/src/layers/MarkerCluster.js:69]: What is `this.model.get('data')`?

**Comment by martinRenou** [comment]
Sorry for not looking into this PR before. Thanks!

File: js/src/layers/MarkerCluster.js (hunk 1 above)
Comment (matyasrada) [reply js/src/layers/MarkerCluster.js]: Some remnant code I believe. It can be deleted. my bad

**Comment by martinRenou** [comment]
Thanks!

**Pull request merged**